### PR TITLE
RFC: homepage field for buildpacks

### PR DESCRIPTION
**Pull request by zmackie**
[readable](https://github.com/zmackie/rfcs/blob/buildpack-links/text/0000-links-for-buildpacks.md)

### CODE REVIEW

File: text/0000-links-for-buildpacks.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- Name: Buildpack homepage
+- Start Date: 1-23-2020
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This RFC suggest a mechanism where buildpacks can provide arbitrary links in their configuration: point to a homepage. These links are intended to contain documentation, point to github repos, point users to readmes, etc. but should not prescribe a format.
```
Comment (nebhale) [inline text/0000-links-for-buildpacks.md]: I'm not sure I love the key `homepage` (can't decide to whether to host mine at Geocities or MySpace) but in the absence of something better, it's good enough.

Comment (jkutner) [reply text/0000-links-for-buildpacks.md]: @nebhale lol, but i agree. That said, there seems to be plenty of prior art.

**Comment by jkutner** [comment]
@zmackie i know the schema is kind of obvious, but would you mind adding an example `buildpack.toml` to the RFC?

File: text/0000-links-for-buildpacks.md (hunk 1 above)
Comment (zmackie) [reply text/0000-links-for-buildpacks.md]: Yeah me too. But old-skool is 😎sometimes.

File: text/0000-links-for-buildpacks.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Too generic a pointer?
+
+# Alternatives
+[alternatives]: #alternatives
+
+- A `docs` field containing markdown.
```
Comment (hone) [inline text/0000-links-for-buildpacks.md]: I like how `homepgae` is generic, but I do wonder if we want a link to docs or maybe add docs in addition.

**Pull request closed without merging**